shared/src/helpers/task.coffee

```diff
@@ -0,0 +1,169 @@
+_ = require 'underscore'
+deepMerge = require 'lodash/merge'
+
+UiSettings = require '../model/ui-settings'
+{_sectionFormat} = require '../components/chapter-section-mixin'
+{
```

Comment:
yah i think a helper for this makes a lot of sense, i just didnt want to refactor that as well in this PR